commands/channel_test.go

```diff
@@ -2546,9 +2549,11 @@ func (s *MmctlUnitTestSuite) TestMoveChannelCmdF() {
 			Times(1)
 
 		err := moveChannelCmdF(s.client, cmd, []string{dstTeamID, channelID})
-		s.Require().Nil(err)
-		s.Len(printer.GetErrorLines(), 1)
-		s.Contains(printer.GetErrorLines()[0], fmt.Sprintf("unable to move channel %q: ", "some-name"))
+		var expected error
+		expected = multierror.Append(expected, errors.New(fmt.Sprintf("unable to move channel %q: some-error", "some-name")))
+
+		s.Require().NotNil(err)
```

Comment:
```suggestion
		s.Require().NoError(err)
```
This applies to similar occurrences.

Reply:
why we are saying no error is going to be returned cause we will receive an error? After changing it to No Error `make test` fails.

Reply:
Ah sorry @s4kh it should be `s.Require().Error(err)`, my bad..